### ISSUE TITLE
Prevent scrolling through sections tmultiple sections are present

### ISSUE THREAD

**Issue by Bondra** [alvarotrigo/fullPage.js#2423]
Hi!

I've encountered a scrolling problem on high resolution displays. When more than one section is present in the viewport, all present sections has to be scrolled through... even though there's nothing to scroll.

Have a look over here for a better understanding:
https://jsfiddle.net/15s3rjjn/6

**Comment by alvarotrigo** [comment]
As detailed in the docs:

> This is ideal for footers. It is important to realise that it doesn't make sense to have all of your sections using this feature. **If there is more than one section in the initial load of the site, the plugin won't scroll at all to see the next one as it will be already in the viewport.**

The issue with this approach of having multiple sections totally visible on the viewport at the same time is that fullPage.js can not know which is the active section at the moment.

- Which anchor should it use in the URL? 
- Which state class should it apply to the body element? 
- Which `afterLoad` callback should it fire?  
- To which section, navigation bullet or menu element should it apply the `active` class? 
- Which media elements should it autoplay /pause if they all contain any?

So, the problem is not in forcing fullPage.js to move to the right section on scroll, but in all it entails regarding the state of the website.

**Comment by alvarotrigo** [comment]
Have you considered using the [Offset Sections](http://alvarotrigo.com/fullPage/extensions/offsetSections.html) extension for it? 
This way no matter the resolution you'll always have the same amount of sections in the site.

**Comment by Bondra** [comment]
alvarotrigo no I haven't. Didn't know it existed but seem to cover exactly the problems I'm having :) Is it a separate plugin that has to be bought? (no problem in supporting just wondering)

**Comment by alvarotrigo** [comment]
> Is it a separate plugin that has to be bought?

It is not a plugin, it is a fullPage.js extension. And yes, it has to be bought but you still need fullPage.js to make it work.
See [how to use extensions](https://github.com/alvarotrigo/fullPage.js#use-extensions) for more info.

**Comment by Bondra** [comment]
Allright so I bought it but wonder what to do when it comes to activation. 
The site is currently in a staging environment with no domain. Will it still work locally if I enter the domain that it will be using later on?

**Comment by alvarotrigo** [comment]
> Will it still work locally if I enter the domain that it will be using later on?

It is easier if you just try it yourself and find it out :)

**Comment by Bondra** [comment]
Yes I know but was just observant with this part "You will only be able to this once according to your license "For up to one website". Since the local version isn't the one that will be used later on I thought it might be stupid of me to enter something that can't be used later :)

**Comment by alvarotrigo** [comment]
Yeah, I'll have to be more specific in the docs about it :)
